src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1611,27 +1619,20 @@ private void embedGenericHandle(IntPtr _this, ref CORINFO_RESOLVED_TOKEN pResolv
                 // TODO? If we're embedding a method handle for a method that points to a sub-class of the actual
                 //       class, we might need to embed the actual declaring type in compileTimeHandle.  
 
+                // TODO: shared generics
                 // IsSharedByGenericInstantiations would not work here. The runtime lookup is required
                 // even for standalone generic variables that show up as __Canon here.
                 //fRuntimeLookup = th.IsCanonicalSubtype();
-            }
 
-            Debug.Assert(pResult.compileTimeHandle != null);
-
-            // TODO: shared generics
```

Comment:
The intention of this was to keep the code layout exactly same as everywhere else we implement the JitInterface...